.github/workflows/npmpublish.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
@AsimAM is this even used? Should we just delete this file?

Reply:
Dosen't look like it, I assume we just kept it/updated it as it was forked.

Reply:
Would be good to delete it 🙂